stackslib/src/chainstate/stacks/boot/pox-4.clar

```diff
@@ -829,7 +845,8 @@
                                    (amount-ustx uint)
                                    (pox-addr { version: (buff 1), hashbytes: (buff 32) })
                                    (start-burn-ht uint)
-                                   (lock-period uint))
+                                   (lock-period uint)
```

Comment:
Same here -- can you add a comment to the method doc about the signing key?

Reply:
Updated a sentence & added the following to comment doc above function:

`;; * You need to provide a signing-key used to allocate signature-weight to a valid signer`